$NetBSD: patch-cdb28383402d248dbc6062f4391b038375c52385-CMakeLists.txt,v 1.1 2021/03/07 10:51:06 kleink Exp $

commit cdb28383402d248dbc6062f4391b038375c52385
Author: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date:   Fri Jul 17 21:25:03 2020 +0200

    CMakeLists.txt: fix paths when FTDIPP is set
    
    Use the same project name (i.e. libftdi1 and not libftdipp1) when FTDIPP
    is enabled as suggested by Aurelien Jarno in
    http://developer.intra2net.com/mailarchive/html/libftdi/2020/msg00044.html
    
    Without this change, the libftdi1.pc config file defines the include
    path as /usr/local/include/libftdipp1 while the ftdi.h file is actually
    installed in /usr/local/include/libftdi1
    
    This is an issue for example for libsigrok which will fail on:
    
    In file included from src/hardware/asix-sigma/protocol.c:27:
    src/hardware/asix-sigma/protocol.h:28:10: fatal error: ftdi.h: No such file or directory
       28 | #include <ftdi.h>
          |          ^~~~~~~~
    
    Fixes:
     - http://autobuild.buildroot.org/results/1427f44e36752c337791597fab47a1889552a2fe
    
    Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>

--- CMakeLists.txt.orig	2020-07-07 19:32:55.000000000 +0000
+++ CMakeLists.txt
@@ -136,7 +136,7 @@ endif ()
 
 add_subdirectory(src)
 if ( FTDIPP )
-  project(libftdipp1 C CXX)
+  project(libftdi1 C CXX)
   add_subdirectory(ftdipp)
 endif ()
 if ( PYTHON_BINDINGS )
